config/config.exs

```diff
@@ -1 +1,14 @@
 use Mix.Config
+
+config :logger, level: :info
+
+config :mini_repo,
```

Comment:
We could read the files inside the module, if you need to create a *mix task* to generate the keys, you can't load in config. because it won't work.

Reply:
I recommend add only the files path here